axes:
  linear_speed: linear
  steering_angle: angular
buttons:
  slow_mode: key_w
  turbo_mode: key_x
